app/views/pages/home.html.erb

```diff
@@ -36,36 +36,37 @@
   </div>
 </div>
 </div>
-<div class="container">
-  <div class="align-center">
-    <h4 class="font-cond-b fg-text-d lts-md fs-300 fs-300-xs no-mg">Why it works</h4>
-  </div>
-  <br>
-  <div class="row">
-    <div class="col">
-     <div class="align-center">
-      <p>"Research has shown
+
+<div class="contener">
+    <div class="align-center why">
+      <h4 class="font-cond-b fg-text-d lts-md fs-300 fs-300-xs no-mg"><span class="text-why">Why it works</span></h4>
+    </div>
+    <div class="contener-fluid why">
+    <div class="row">
+      <div class="col">
+        <div class="align-center">
+        <p>Research has shown
         that if you make a conscious decision that you want to achieve something
         and commit to someone that you will do it, there’s a 65% chance of success.
-        The success rate increases to 95% if you report back on your progress to
```

Comment:
Hi Juli, I think we need to keep 95%, it shows the importance of our App. We can think of re-phrasing if it does not fit the container. But for me it is the most important phrase in the whole "Why" section